app/components/chat_messages_group/chat_messages_group.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="ChatMessagesGroup">
+  <div class="ChatMessagesGroup-header">
+    <div class="ChatMessagesGroup-avatar">
+      <%= c 'avatar', user: user, styles: [:small] %>
+    </div>
+
+    <%= c 'heading', tag: :h3 do %>
+      <a class="ChatMessagesGroup-user" href="<%= user_path(user) %>">
+        <%= user.name %>
+      </a>
+    <% end %>
+
+    <div class="ChatMessagesGroup-actions">
+      <a href="<%= conversation_link %>">
+        <%= c 'icon', icon: 'reply-arrow', styles: [:inline] %> nachfragen
```

Comment:
Use `I18n` please.

Reply:
This button is really tiny and I wouldn't have seen it.

Reply:
Good catch, I think I messed that up when rebasing.